internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -134,7 +134,34 @@ func (ms *metadataStorage) store(ctx context.Context) (tableName string, created
 		return
 	}
 
-	tableName = collectionNameToTableName(ms.collection)
+	// Index to ensure that table name is unique
+	key = IndexKey{{Field: `table`, Order: types.Ascending}}
+	if err = createPgIndexIfNotExists(ctx, ms.tx, ms.db, dbMetadataTableName, dbMetadataIndexName, key, true); err != nil {
+		err = lazyerrors.Error(err)
+		return
+	}
```

Comment:
Let's write it down in the function's comment

Reply:
Added